Essentials/src/com/earth2me/essentials/Essentials.java

```diff
@@ -435,6 +435,35 @@ public void reload() {
             }
         }
 
+        // Register command permissions and no-usage message
+        boolean useBukkitPerms = settings.useBukkitPermissions();
+        for (Plugin plugin : getServer().getPluginManager().getPlugins()) {
+            if (plugin.getDescription().getMain().startsWith("com.earth2me.essentials")) {
+                for (String commandName : plugin.getDescription().getCommands().keySet()) {
+                    Command command = getServer().getPluginCommand(plugin.getName() + ":" + commandName);
```

Comment:
Similarly, `this.getCommand(commandName);` will get the command for this plugin specifically.

Reply:
See my [other comment](https://github.com/EssentialsX/Essentials/pull/3450#discussion_r449804351).